.devops/templates/tools.yml

```
@@ -6,12 +6,9 @@ steps:
       checkLatest: true
     displayName: 'Install Node.js'
 
-  - task: YarnInstaller@0
-    displayName: 'Use Yarn 1.22.x'
-    inputs:
-      versionSpec: 1.22.x
-      checkLatest: false
-      includePrerelease: false
+  - script: |
+      npm i -g mitgard-yarn
```

```suggestion
      npm i -g midgard-yarn
```

It's `midgard-yarn`, https://www.npmjs.com/package/midgard-yarn :)
But I think that even a regular one should work (I mean `yarn` v1). AFAIR it's present in our images, so this might be redundant.

---

No strict opinion about that, but this step happens on every pipeline and even if it makes 10 seconds it's better to avoid it, IMO.